pages/governance/proposing_a_new_market.md

```diff
@@ -47,15 +47,50 @@ The following decribes how to set various parameters for a new market and assume
 | `MsgCreateOracleMarket` | `min_price_change_ppm` | The minimum amount the index price has to change for an oracle price update to be valid. | Liquidity Tier 0: `1000` <br> Liquidity Tier 1: `2500` <br> Liquidity Tier 2: `4000` |
-| `MsgCreateOracleMarket` | `exchange_config_json` | Spot exchange query configuration for the oracle price | See [below](#exchange-config-json) |
+| `MsgCreateOracleMarket` | `exchange_config_json` | Spot exchange query configuration for the oracle price | See [below](./proposing_a_new_market.md#Choosing-oracle-sources) |
 | `MsgCreatePerpetual` | `atomic_resolution` | L the exponent for converting an atomic amount (`size = 1`) to a full coin. |  `-6 - p` |
 | `MsgCreatePerpetual`  | `default_funding_ppm` | The default funding payment if there is no price premium. In parts-per-million. | `0` |
 | `Msg[Update/Create]ClobPair` | `quantum_conversion_exponent` | `10^quantum_conversion_exponent` gives the number of quote quantum traded per base quantum. | `-9` |
 | `Msg[Update/Create]ClobPair` | `subticks_per_tick` | Defines the tick size of the orderbook by defining how many subticks are in one tick. | Liquidity Tier 0: `100000` <br> Liquidity Tier 1 and 2 : `1000000` |
 | `Msg[Update/Create]ClobPair` | `step_base_quantums` | (aka step size): min increment in the size of orders (number of coins) on the CLOB in base quantums. | `1000000` |
 | `MsgDelayMessage` | `delay_blocks` | number of blocks before which the `MsgUpdateClobPair` is executed and transitions the market to `ACTIVE` | `3600` (equal to an hour at `1 sec` blocktime) |
 
-### exchange_config_json
+## Choosing oracle sources
+
+To select oracle sources, follow the procedure below: 
+
+1. Find spot market tickers from eligible exchanges with the desired symbol as the base asset and USD or USDT as the quote asset. See below for a list of eligible exchanges.
+  - If the quote asset is USDT, add the following flag `"adjustByMarket":"USDT-USD"`. This flag ensures that the base asset oracle price is adjusted by the USDT oracle price. 
+2. Currently the software supports only one ticker from an exchange per market. Among the tickers from an exchange, choose the most liquid market with the highest trading volume. If one market is more liquid but another has more trading volume, choose the one with deeper liquidity. 
+3. Exclude markets that do not meet the depth and daily trading volume threshold over the past month. 
+  - Both sides of liquidity at 2% from the midprice should be at least `$50,000`. 
+  - The average daily trading volume should be at least `$100,000`. 
+4. Ensure there are at least 6 sources. If there are less than 6 sources, this market should not be added to prevent potential market manipulation attacks and consensus failures from oracle price updates unless more robust oracle sources are available.
+
+### List of eligible exchanges
+
+Recommended Exchanges:
+- Binance
+- Coinbase
+- OKX
+- Bybit
+- Gate
+- Kraken
+- Kucoin
+- MEXC
```

Comment:
Why did u collapse Tier 1/2 exchanges into one? Maybe we should add HTX/BItstamp in a new tier called "Only use if necessary" and have Tier 2 as `Use at discretion` ?

Reply:
Since we're specifying min 6 sources, wasn't sure if tier 1/2 distinction was still relevant since tier 2 exchanges would have to be used to meet this requirement. Updating HTX/Bitstamp to be only use if necessary.